src/app/users/users.service.ts

```diff
@@ -24,13 +24,10 @@ export class UsersService {
 
   async generateUserCredentials(member: Member) {
     const password = await this.passwordService.hashPassword(uuidv4());
-    const userNumber = Math.floor(Math.random() * 10000);
-    const userName = `${member.firstName.replace(" ", "-")}_${
+    const count = await this.prisma.user.count();
```

Comment:
The count here is total number of users. Should this be the count of people having same usernames instead?

Reply:
This will probably be a design choice, so let's discuss this in call.

Reply:
forgot to discuss this last time, lets discuss next time